chaosaws/asg/actions.py

```diff
@@ -343,12 +343,17 @@ def detach_random_instances(asg_names: List[str] = None,
                 'than exist on asg %s' % (a['AutoScalingGroupName']))
 
         instances = random.sample(instances, instance_count)
+        instances = sorted(instances)
 
         response = client.detach_instances(
             AutoScalingGroupName=a['AutoScalingGroupName'],
-            InstanceIds=sorted(instances),
+            InstanceIds=instances,
```

Comment:
I'm curious why this was removed?

Reply:
I put this in the previous statement to reuse it in the return value, which is then used in the tests.

Reply:
I had missed that. Good one.